server/sonar-server/src/main/java/org/sonar/server/computation/measure/MeasureVariations.java

```diff
@@ -50,14 +56,42 @@ private Builder() {
       // prevents instantiation outside static method
     }
 
+    /**
+     * @deprecated use {@link #setVariation(Period, double)} instead
+     */
+    @Deprecated
     public Builder setVariation(int index, double variation) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)